buildenv

```diff
@@ -1,9 +1,9 @@
 export ZOPEN_TYPE="GIT"
 export ZOPEN_CONFIGURE="./Configure"
-export ZOPEN_CONFIGURE_OPTS="-Dprefix=\${ZOPEN_INSTALL_DIR} -Duserelocatableinc -Dusedevel -des -Duse64bitall -Dusedl"
+export ZOPEN_CONFIGURE_OPTS="-Dprefix=\${ZOPEN_INSTALL_DIR} -Duserelocatableinc -Dusedevel -des -Duse64bitall -Dusedl -D ccflags=\"\$CFLAGS \$CPPFLAGS -DOS390 -DZOS -DNO_LOCALE_MESSAGES -DOEMVS -DMAXSIG=39 -qNOOPT\" -D ldflags=\"\$LDFLAGS\" -D libs=\"\$LIBS\""
```

Comment:
We have to drop to no opt ?

Reply:
Yes, opened issue: https://github.com/ZOSOpenTools/perlport/issues/60

Turns out we never built it with -O2 or above, see this in the hints/os390.sh:
```
  119 # Turning on optimization causes perl to not even compile from miniperl.  You
  120 # can override this with Configure -Doptimize='-O2' or somesuch.
  121 case "$optimize" in
  122 '') optimize=' ' ;;
  123 esac
```